presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -129,6 +131,13 @@
     private FunctionAssertions functionAssertions;
     private List<ListenableFuture<Void>> futures;
 
+    private static <T> T[] concat(T[] first, T[] second)
```

Comment:
Line 93. I separated list of longs used for tests into small longs and extreme longs (e.g. `Long.MAX_VALUE`) which may cause overflow while multiplied/added etc.
It's just not to affect other tests than `testBinaryOperatorsLongLong` and not to duplicate code (for other tests I keep the same list as before).